<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="addRoomBtn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>98</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>添加房间</string>
    </property>
   </widget>
   <widget class="QLabel" name="roomLab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>房间号</string>
    </property>
   </widget>
   <widget class="QLabel" name="pwdLab">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roomText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pwdText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">客户端列表</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>83</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="realTem">
       <property name="text">
        <string>实时温度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="realTemNum">
       <property name="digitCount">
        <number>3</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>83</width>
      <height>45</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="moneyLab">
       <property name="text">
        <string>金钱</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="money">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>83</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="speed">
       <property name="text">
        <string>风速</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="speedNum">
       <property name="digitCount">
        <number>3</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>83</width>
      <height>45</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="modeLab">
       <property name="text">
        <string>模式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mode">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>83</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="setTem">
       <property name="text">
        <string>设置温度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="setTemNum">
       <property name="digitCount">
        <number>3</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>83</width>
      <height>45</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="powerLab">
       <property name="text">
        <string>功耗</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="power">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="heatModeBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>98</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>制暖</string>
    </property>
   </widget>
   <widget class="QPushButton" name="coldModeBtn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>98</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>制冷</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>98</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="TempLab">
       <property name="text">
        <string>中央空调温度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="tempBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>98</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLabel" name="flashSpdLab">
       <property name="text">
        <string>刷新频率</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="flashSpdBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
